File: client/butterfly/app/src/main/java/pf/com/butterfly/message/Protocols/register_req.java
package pf.com.butterfly.message.Protocols;
import android.util.Log;
import org.json.JSONArray;
import org.json.JSONObject;
import pf.com.butterfly.message.MsgBase;
import pf.com.butterfly.message.MsgCodeId;
  public class register_req extends MsgBase
  {
      public register_req()
      {
          CodeId = MsgCodeId.register_req;
      }
      public String tel;    ////手机号
      public String nickname;    ////昵称
      public String pass;    ////密码

}

File: server/hudie/hudie/message/Protocols/deng_lu/register_req.cs
using messages;

namespace messages.Protocols
{
  public class register_req : MsgBase
  {
      public register_req()
      {
          CodeId = MsgCodeId.register_req;
      }
      public string tel;    ////手机号
      public string nickname;    ////昵称
      public string pass;    ////密码

	}
}
